src/pages/community-champions.js

```diff
@@ -541,6 +541,7 @@ const PinLocationImage = styled('img')`
 
 const LocationText = styled(P4)`
     color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    white-space: nowrap;
```

Comment:
using this so location text can stay on one line. some locations exceed the content width and then wrap:
![Screen Shot 2021-07-14 at 3 14 56 PM](https://user-images.githubusercontent.com/46455728/125679977-67626c5a-f5da-4849-add2-91c1019288bc.png)

perhaps we should limit the number of characters a location can be...

Reply:
nice! Might be a good Mike question but I don't think locations will be longer than this